compose-files/docker-compose.yml

```
@@ -21,7 +21,7 @@ services:
   ### Schul-Cloud Server
   ###
   mongodb-primary:
-    image: mongo:5.0.26 ## change tag in version.env
+    image: mongo:6.0.15
```

Last version after docker hub is 6.0.16 
```suggestion
    image: mongo:6.0.16
```